stl/inc/condition_variable

```diff
@@ -40,48 +39,68 @@ public:
     condition_variable_any& operator=(const condition_variable_any&) = delete;
 
     void notify_one() noexcept { // wake up one waiter
-        lock_guard<mutex> _Lck(*_Myptr);
-        _Check_C_return(_Cnd_signal(_Mycnd()));
+        const shared_ptr<mutex> _Ptr = _Myptr; // for immunity to *this destruction

```

Comment:
In the interest of getting this PR applied, let's pull this alleged race fix out into a separate PR. (And on 48.)

Reply:
This is #883.